### PR TITLE
Fix TemplateSpatialModel lon_0 and lat_0 default

### PR DESCRIPTION
**Pull request by QRemy**
Fix TemplateSpatialModel lon_0 and lat_0 default

### CODE REVIEW

**Review by AtreyeeS — COMMENTED** [review]
Thanks @QRemy ! I suppose this should not affect the tests because in practice the default values are the centre of the map?

**Comment by QRemy** [comment]
That affects from_fict when the values are not given in the dict,  I will add a test for that.

**Comment by bkhelifi** [comment]
Can you precise this behaviour in the docstring? I think that this is important for the users

**Comment by QRemy** [comment]
added test and more details in the docstring

File: gammapy/modeling/models/spatial.py [hunk 1]
```diff
@@ -1257,7 +1257,8 @@ class TemplateSpatialModel(SpatialModel):
     """Spatial sky map template model.
 
     For more information see :ref:`template-spatial-model`.
-    The center of the model position can be fit on the data.
+    By default the position of the model is fixed at the center of the map.
+    The position can be fittted unfreezing the `lon_0` and `lat_0 `parameters.
```
Comment (AtreyeeS) [inline gammapy/modeling/models/spatial.py:1261]:
```suggestion
    The position can be fittted by unfreezing the `lon_0` and `lat_0 `parameters.
```

**Review by AtreyeeS — APPROVED** [review]
Thanks for the fix @QRemy !

**Pull request merged**